docs/best-practices.md

```diff
@@ -840,20 +840,22 @@ pod will end up on specific node according your requirements like operating syst
-| `kube-state-metrics`    | `kube-prometheus-stack.kube-state-metrics.nodeSelector.kubernetes.io/os`        |
-| `prometheus`            | `kube-prometheus-stack.prometheus.prometheusSpec.nodeSelector.kubernetes.io/os` |
-| `otelagent`             | `otelagent.daemonset.nodeSelector.kubernetes.io/os`                             |
-| `otelcol`               | `otelcol.deployment.nodeSelector.kubernetes.io/os`                              |
-| `otelgateway`           | `otelgateway.deployment.nodeSelector.kubernetes.io/os`                          |
-| `otellogs`              | `otellogs.daemonset.nodeSelector.kubernetes.io/os`                              |
-| `metadata`              | `metadata.metrics.statefulset.nodeSelector.kubernetes.io/os`                    |
-| `metadata`              | `metadata.logs.statefulset.nodeSelector.kubernetes.io/os`                       |
-| `pvcCleaner`            | `pvcCleaner.job.nodeSelector.kubernetes.io/os`                                  |
-| `metrics otelcol`       | `sumologic.metrics.collector.otelcol.nodeSelector.kubernetes.io/os`             |
+| component                | key                                                                             |
+| ------------------------ | ------------------------------------------------------------------------------- |
+| `sumologic`              | `sumologic.setup.job.nodeSelector.kubernetes.io/os`                             |
+| `kube-prometheus-stack`  | `kube-prometheus-stack.prometheus-node-exporter.nodeSelector.kubernetes.io/os`  |
+| `kube-state-metrics`     | `kube-prometheus-stack.kube-state-metrics.nodeSelector.kubernetes.io/os`        |
+| `prometheus`             | `kube-prometheus-stack.prometheus.prometheusSpec.nodeSelector.kubernetes.io/os` |
+| `remoteWriteProxy`       | `sumologic.metrics.remoteWriteProxy.nodeSelector.kubernetes.io/os`              |
+| `otelcolInstrumentation` | `otelcolInstrumentation.statefulset.nodeSelector.kubernetes.io/os`              |
+| `tracesGateway`          | `tracesGateway.deployment.nodeSelector.kubernetes.io/os`                        |
+| `otellogs`               | `otellogs.daemonset.nodeSelector.kubernetes.io/os`                              |
+| `setupJob`               | `sumologic.setup.job.nodeSelector.kubernetes.io/os`                             |
+| `tracesSampler`          | `tracesSampler.deployment.nodeSelector.kubernetes.io/os`                        |
+| `metadata`               | `metadata.metrics.statefulset.nodeSelector.kubernetes.io/os`                    |
+| `metadata`               | `metadata.logs.statefulset.nodeSelector.kubernetes.io/os`                       |
+| `pvcCleaner`             | `pvcCleaner.job.nodeSelector.kubernetes.io/os`                                  |
+| `metrics otelcol`        | `sumologic.metrics.collector.otelcol.nodeSelector.kubernetes.io/os`             |
```

Comment:
I think we should split this into two tables:
1. Just the `sumologic.nodeSelector` and subchart options
2. The component specific options

While we're at it, let's also add missing subcharts here: otel-operator, telegraf-operator and falco.

Reply:
note, we don't have `telegraf-operator` listed [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/deploy/helm/sumologic#configuration), I'll try to drop another pr for that late